packages/kununu-utils/kununu-helpers/formValidation/index.js

```diff
@@ -46,10 +46,30 @@ export function isEmail (value) {
   return undefined;
 }
 
+/**
```

Comment:
Please write a test for `isRegexMatch`, other validations (`isEmpty`, `minLength`, `isEmail`, ) are covered in `formValidation/index.spec.js`